### PR TITLE
STUTL-6 add tests

### PR DESCRIPTION
**Pull request by zburke**
Provide tests for functions in `lib`.

Refs [STUTL-6](https://issues.folio.org/browse/STUTL-6)

### CODE REVIEW

**Review by rjruizes — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]
Nice setup @zburke!

**Review by Baroquem — APPROVED** [review]

**Review by annamelnyk — APPROVED** [review]

**Comment by annamelnyk** [comment]
Great work!
I am wondering why Sonar does not count coverage? Should it have special setup for Jest?

**Comment by zburke** [comment]
I'm not sure why SonarCloud can't find the coverage report, @annamelnyk. I asked @funkymalc about that and he's looking into it. I copied the config over from ui-data-import so I hoped it might just work 🤞 but wasn't so lucky.

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const esModules = ['@folio'].join('|');
+
+module.exports = {
+  collectCoverageFrom: [
+    '**/(lib|src)/**/*.{js,jsx}',
+    '!**/node_modules/**',
+    '!**/test/**',
+  ],
+  coverageDirectory: './artifacts/coverage/',
```
Comment (aliaksei-chumakou) [inline jest.config.js:12]:
```suggestion
  coverageDirectory: './artifacts/coverage-jest/',
```
https://github.com/folio-org/stripes-acq-components/blob/master/jest.config.js#L12

File: lib/exportCsv.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import exportCsv from './exportCsv';
+
+const list = [
+  { a: 'r1a', b: 'r1b', c: 'r1c' },
+  { a: 'r2a', b: 'r2b', c: 'r2c' },
+  { a: 'r3a', b: 'r3b', c: 'r3c' },
+];
+
+describe.only('download a CSV file', () => {
```
Comment (aliaksei-chumakou) [inline lib/exportCsv.test.js:11]: `only`?

File: jest.config.js (hunk 1 above)
Comment (aliaksei-chumakou) [reply jest.config.js:12]: cc @funkymalc is it right to use in every module? I'm not sue that sonarcloud fetches coverage from there

**Pull request merged**